src/ontology/config/external-content-robot-headers.json

```diff
@@ -0,0 +1,11 @@
+{
+    "mondo_id": "ID",
+    "report_ref": "A oboInOwl:hasDbXref",
+    "report_ref_source": ">A oboInOwl:source",	
+    "preferred_name": "A oboInOwl:hasExactSynonym",
+    "preferred_name_source": ">A oboInOwl:hasDbXref",	
+    "synonym_type": ">A oboInOwl:hasSynonymType",	
+    "subset": "AI oboInOwl:inSubset",	
```

Comment:
`AI`???

Reply:
An annotation to an IRI: http://robot.obolibrary.org/template#generic-template-strings

If it's just A, the expectation is that the value is a literal.